WORKSPACE.bazel

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A workspace for gapic-showcase
+"""
+workspace(name = "gapic_showcase")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+##
+# googleapis for the common protos
+#
+http_archive(
+  name = "com_google_googleapis",
+  strip_prefix = "googleapis-cf41866c6f14f10a07aa1e2a1260fc0a2727d889",
+  urls = ["https://github.com/googleapis/googleapis/archive/cf41866c6f14f10a07aa1e2a1260fc0a2727d889.zip"],
+  sha256 = "943602b1e91621f282c1e790eb46d9a70e202fe65091b7d41415a43973cebec4"
+)
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
+switched_rules_by_language(name = "com_google_googleapis_imports", grpc = True)
+
+##
+# protobuf for the protobuf protos
+#
+http_archive(
```

Comment:
Can we get rid of `gapic-showcase/schema/api-common-protos/WORKSPACE` which defines essentially the same dependency (but older version) so we don't have nested workspaces?

Reply:
The `gapic-showcase/schema/api-common-protos` is another repository (https://github.com/googleapis/api-common-protos) brought by submodule so I don't think we **want** to change anything there.

Nested workspaces however are not an issue, the `bazel_example` in `gapic-generator-ruby` is a nested workspace (https://github.com/googleapis/gapic-generator-ruby/blob/master/bazel_example/WORKSPACE.bazel); it is a supported scenario.

Reply:
Oh, I see. I misread that the `urls` line was referring to similar things. My bad.

I'm OK with this. Do please get sign-off from @noahdietz as well, though.